05 - HTML/README.md

```diff
@@ -1 +1,437 @@
+
+The `table` tag uses several tags inside, such as `thead`, `tbody`, `th`, `tr` and `td`. `thead` is used to create a header for the table, like adding a title for every column, inside `tbody` we will display all the data we have. `tr` stands for **table row**, which indicate a row for the table and `td` means **table data** and goes inside the `tr`.
+
+```<table>
+    <thead>
+        <th>First column</th>
+        <th>Second column</th>
+    </thead>
+    <tbody>
+        <tr>
+            <td>A</td>
+            <td>B</td>
+        </tr>
+        <tr>
+            <td>C</td>
+            <td>D</td>
+        </tr>
+        <tr>
+            <td>E</td>
+            <td>F</td>
+        </tr>
+    </tbody>
+</table>
+```
+This is the result from above code:
+
+<table>
+    <thead>
+        <th>First column</th>
+        <th>Second column</th>
+    </thead>
+    <tbody>
+        <tr>
+            <td>A</td>
+            <td>B</td>
+        </tr>
+        <tr>
+            <td>C</td>
+            <td>D</td>
+        </tr>
+        <tr>
+            <td>E</td>
+            <td>F</td>
+        </tr>
+    </tbody>
+</table>
+
+If your table doesn't require a `thead` you can simply start declaring rows directly after the `table` tag, like this:
+
+<table>
+    <tr>
+        <td>A</td>
+        <td>B</td>
+    </tr>
+    <tr>
+        <td>C</td>
+        <td>D</td>
+    </tr>
+    <tr>
+        <td>E</td>
+        <td>F</td>
+    </tr>
+</table>
+
+You can also merge columns or rows using the `colspan` and `rowspan` attributes:
+
+```
+<table>
+    <tr>
+        <td>A</td>
+        <td>B</td>
+    </tr>
+    <tr>
+        <td colspan="2">C</td>
+    </tr>
+    <tr>
+        <td>D</td>
+        <td>E</td>
+    </tr>
+</table>
+```
+<table>
+    <tr>
+        <td>A</td>
+        <td>B</td>
+    </tr>
+    <tr>
+        <td colspan="2">C</td>
+    </tr>
+    <tr>
+        <td>D</td>
+        <td>E</td>
+    </tr>
+</table>
+
+```
+<table>
+    <tr>
+        <td rowspan="2">A</td>
+        <td>B</td>
+    </tr>
+    <tr>
+        <td>C</td>
+    </tr>
+    <tr>
+        <td>D</td>
+        <td>E</td>
+    </tr>
+</table>
+```
+
+<table>
+    <tr>
+        <td rowspan="2">A</td>
+        <td>B</td>
+    </tr>
+    <tr>
+        <td>C</td>
+    </tr>
+    <tr>
+        <td>D</td>
+        <td>E</td>
+    </tr>
+</table>
+
+## Forms
+
+Forms are one of the most important elements in web applications, these are the way that users can interact with the data, by creating, updating or even deleting data from their records. In social networks are the way you post status, upload your photos, create an account, do log in into the application, etc.
```

Comment:
`In social networks are the way you post status`

could be?

`In social networks that's the way you post a status`